go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/distribution/distribution/v3
 
-go 1.23.7
+go 1.23.10
```

Comment:
Yeah, I think we can keep this one at either 1.23.7 (as there's no need to update the minimum version), or if possible, ideally just set it to 1.23.0 (but it's possible that modules we depend on did update to a higher version).

Reply:
I've just tried 1.23.0 works just fine.